### DISCUSSION TITLE
Improve with 2 * DHCP

### DISCUSSION THREAD

**Discussion by F5BJR** [ipxe/ipxe#380] [Q&A] [answered]
It's not that serious 

I cannot touch the network and I have two dhcp : OpenWrt with **iPXE** ( wan not used ) & multiwan SRX5308 
I need the router to launch debian for example with an Internet installation without plugging and re-plugging cables

iPXE works purpose:
when i start the target iPXE pc it doesn't work
I have to restart a second time for the iPXE menu to appear

would there be a trick to get this started at the first time ?

![iPXE](https://user-images.githubusercontent.com/32757143/121260595-d13af280-c8b1-11eb-9c63-f8fea7ecf254.jpg)

**Comment by mcb30** [comment, marked as answer]
If you have two DHCP servers on the network providing conflicting information, then you have a misconfigured network.  It will be largely random which DHCP server ends up getting used by iPXE.  This is as per the design of DHCP.

**Reply by F5BJR** [reply]
I have add opkg install dhcp-forwarder 

dnsmasq.conf with this line  👍 

dhcp-relay=192.16.0.241,192.168.0.1

and run without problem
